src/content/reference/react/useTransition.md

```diff
@@ -409,6 +430,447 @@ b { display: inline-block; margin-right: 10px; }
 
 ---
 
+### Creating an Action with async Transitions {/*creating-an-action-with-async-transitions*/}
+
+<NextMajor>
+
+Async transitions are currently available in React 19 beta, and the latest React Canary.
+
+</NextMajor>
+
+Async transitions allow you to submit async requests within Transitions to handle errors, show pending states, and [prevent unwanted loading indicators](#preventing-unwanted-loading-indicators). Async transitions also integrate into features like `useOptimistic` and `<form>` actions. By convention, functions that use async transitions are called "Actions".
+
+You can create an Action by passing an async function to `startTransition`:
+
+```js {8,13}
+import {updateQuantity} from './api';
+
+function CheckoutForm() {
+  const [isPending, startTransition] = useTransition();
+  const [quantity, setQuantity] = useState(1);
+
+  function updateQuantityAction(newQuantity) {
+    startTransition(async () => {
+      const savedQuantity = await updateQuantity(newQuantity);
+      startTransition(() => {
+        setQuantity(savedQuantity);
+      });
```

Comment:
It's probably useful to add a small note in this sandbox that explains why we need the extra `startTransition` here. I was looking for this info but it only appears later in the troubleshooting section.

I also tried to remove the extra `startTransition()` to see if I got a bug, but in this case I didn't really see any behavior change. It's probably only due to that specific example since `setQuantity` happens last